src/retrieve-validation/node.ts

```diff
@@ -1,28 +1,51 @@
 // Imports
 import * as https from 'https';
-import { W3CCSSValidatorResponse } from '.';
+import retrieveValidation, { W3CCSSValidatorResponse } from '.';
 import BadStatusError from './bad-status-error';
+import FormData from 'form-data';
+import { W3CValidatorParameters } from '../types/parameters';
 
 // Utility function for retrieving response from W3C CSS Validator in a Node.js environment
-const retrieveInNode = async (url: string, timeout: number): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
+const retrieveInNode: typeof retrieveValidation = async (
+	method: string,
```

Comment:
This should be more narrow:

```suggestion
	method: 'GET' | 'POST',
```